main.js

```diff
@@ -19,15 +19,19 @@ function Hoverable() {
 	var win = window;
 
 	function init() {
-		win.document.body.setAttribute('data-o-hoverable--js', '');
+		win.document.documentElement.setAttribute('data-o-hoverable--js', '');
 		touchSupport();
 	}
 
+	function updateClasses(action) {
```

Comment:
As they're carried out synchronously with no call inbetween the two to anything that would cause a reflow I'm pretty sure there is only one reflow. (cf when using css animation it's sometimes necessary to explicitly call `el.offsetWidth` after changing a class in order to force a redraw). Besides, for the vast majority of devices the class would change a maximum of once per page view.

I wouldn't say use cases are limited. It is annoying and sometimes confusing when hover effects persist after click on touch devices, though there may be a better solution.